src/device/mod.rs

```diff
@@ -281,6 +283,7 @@ pub struct EventListeners {
     derp_event_subscriber: mc_chan::Rx<Box<DerpServer>>,
     endpoint_upgrade_event_subscriber: chan::Rx<UpgradeRequestChangeEvent>,
     stun_server_subscriber: chan::Rx<Option<StunServer>>,
+    post_quantum: chan::Rx<telio_pq::Event>,
```

Comment:
```suggestion
    post_quantum_subscriber: chan::Rx<telio_pq::Event>,
```

Maybe lets name it a subscriber, so it more clearly states that this is not the post_quantum entity itself?

Reply:
Done